src/app/pages/feed/feed.component.scss

```
@@ -1,7 +1,7 @@
 @import "/src/styles/colors";
 
 @mixin borders {
-  border: 1px solid black;
+  border: 2px solid $border;;
```

remove extra semicolon.
```suggestion
  border: 2px solid $border;
```